WebGoatCore/ApiResponse.cs

```diff
@@ -0,0 +1,44 @@
+using Newtonsoft.Json;
+
+public class ApiResponse
+{
+    public int StatusCode { get; }
+
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public string Message { get; }
+
+    public ApiResponse(int statusCode, string message = null)
+    {
+        StatusCode = statusCode;
+        Message = message ?? GetDefaultMessageForStatusCode(statusCode);
+    }
+
+    private static string GetDefaultMessageForStatusCode(int statusCode)

```

Comment:
Maybe we could use https://stackoverflow.com/questions/43783402/get-http-status-code-descriptions-in-asp-net-core for this?